components/fxa-client/src/mozilla.appservices.fxaclient.protobuf.rs

```diff
@@ -0,0 +1,181 @@
+#[derive(Clone, PartialEq, ::prost::Message)]
```

Comment:
Any chance these can get a `// DO NOT EDIT this file was automatically generated` header on these or similar?

Reply:
Good idea, opened https://github.com/danburkert/prost/pull/300.